src/ResourceManager/IotHub/Commands.IotHub/help/Update-AzureRmIotHub.md

```diff
@@ -0,0 +1,163 @@
+﻿---
+external help file: Microsoft.Azure.Commands.IotHub.dll-Help.xml
+Module Name: AzureRM.IotHub
+online version: 
```

Comment:
online version should be: https://docs.microsoft.com/en-us/powershell/module/azurerm.iothub/update-azurermiothub

Reply:
Added